seleniumByTechnocredits/src/seleniumPractise/amarja/Homework1.java

```diff
@@ -0,0 +1,44 @@
+package seleniumPractise.amarja;
+
+/*Program to automated Alert demo and accept the alert psot valdiation
+1) Alert Demo > Submit > Alert
+Validation of the values of Alert Demo with of the Alert.*/
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class Homework1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/index.html#");
+		driver.findElement(By.partialLinkText("Elements")).click();
+		Thread.sleep(1000);
+		driver.findElement(By.xpath("//input[contains(@id,'UserFirst')]")).sendKeys("Amarja");
```

Comment:
Instead of hardcode value,Please use properties file.

Reply:
Removed hard coded values,Included code to read values from property files